# Compendium of available lists of ligand-receptor pairs in literature

This compendium has been created to collect lists of ligand-receptor pairs
and interactions for surface-secreted proteins that are available in literature.

## Human Lists

- [Kirouac et al. (2010) Dynamic interaction networks in a hierarchically organized tissue](https://doi.org/10.1038/msb.2010.71)
- [Qiao et al. (2014) Intercellular network structure and regulatory motifs in the human
 hematopoietic system](https://doi.org/10.15252/msb.20145141)
- [Ramilowski et al. (2015) A draft network of ligandâ€“receptor-mediated multicellular
 signalling in human](https://doi.org/10.1038/ncomms8866)
- [Choi et al. (2015) Transcriptome Analysis of Individual Stromal Cell Populations
 Identifies Stroma-Tumor Crosstalk in Mouse Lung Cancer Model](https://doi.org/10.1016/j.celrep.2015.01.040)
- [Pavlicev et al. (2017) Single-cell transcriptomics of the human placenta: inferring
 the cell communication network of the maternal-fetal interface](https://doi.org/10.1101/gr.207597.116)
- [Ximerakis et al. (2019) Single-cell transcriptomic profiling of
 the aging mouse brain](https://doi.org/10.1038/s41593-019-0491-3)

## Mouse Lists

- [Ding et al. (2016) A cell-type-resolved liver proteome](https://doi.org/10.1074/mcp.M116.060145)
- [Yuzwa et al. (2016) Proneurogenic ligands defined by
 modeling developing cortex growth factor communication networks](https://doi.org/10.1016/j.neuron.2016.07.037)
- [Skelly et al. (2018) Single-cell transcriptional profiling reveals cellular
 diversity and intercommunication in the mouse heart](https://doi.org/10.1016/j.celrep.2017.12.072)
- [Baccin et al. (2020) Combined single-cell and spatial transcriptomics reveal the molecular,
 cellular and spatial bone marrow niche organization](https://doi.org/10.1038/s41556-019-0439-6)

## Other organisms
